Changelog

```diff
@@ -16,10 +16,23 @@ Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
     - FEATURE: Add option to name each qbittorrent instance (Chocobo1)
     - FEATURE: Add button for sending test email (Thomas Piccirello)
     - FEATURE: Allow torrents to override default share limit action (glassez)
+    - FEATURE: Use Start/Stop instead of Resume/Pause (thalieht) 5d1c2496063b41874e85af0145b153e006fc92d3
+    - FEATURE: Add the Popularity metric (Aliaksei Urbanski) f37d0c486c36e9f6c6e62378385e639feada3085
+    - FEATURE: Focus on Download button if torrent link retrieved from the clipboard (glassez) 7310eec74e6694342e9934ecc524df38d0064bb5
+    - FEATURE: Add ability to pause/resume entire BitTorrent session (glassez) 8ef7d3ec9a56e7445948c37a282629ca5ed551ab
+    - FEATURE: Add an option to set BitTorrent session shutdown timeout (glassez) 2b728b3bc0c525905ef4c6086bc45027eb2150fd
+    - FEATURE: Apply filename filter to subfolder names as well (glassez) d89f289f82fb6ff4305841c7d4c7899ffe12f896
+    - FEATURE: Allow to use regular expression to filter torrent content (glassez) 9894f654cf9f9e5ece5e83b55b709c70e398dd02
     - BUGFIX: Display error message when unrecoverable error occurred (glassez)
     - BUGFIX: Update size of selected files when selection is changed (glassez)
     - BUGFIX: Normalize tags by trimming leading/trailing whitespace (glassez)
     - BUGFIX: Correctly handle share limits in torrent options dialog (glassez)
+    - BUGFIX: Adjust tracker tier when adding additional trackers (Chocobo1) 321d7e5b17b6e4cacf713b4f4409364a5e65873e
+    - BUGFIX: Fix inconsistent naming between (Done/Progress) column (luzpaz) d3315f7cc7f31b1578cf72f07b175a80c204dac9
+    - BUGFIX: Sanitize peer client names (Hanabishi) 2c47f09d7a797c1ea886879749b71198e449e701
+    - BUGFIX: Apply share limits when torrent downloading is finished (glassez) 65d143d4c411e045b3cd1cf847dee831290af799
+    - BUGFIX: Revise Protocol column (Chocobo1) 1c49e0973c94b32bf3bfe52845b6b9572c997ef2
```

Comment:
No protocol column in 4.6.x so mentioning this is meaningless.

Reply:
It's reasonable.
But it is also reasonable to mention fixes from the previous public release of v5.0.x, which is v5.0beta1.
So I would mention all of them in the Changelog, both v5.0beta1 and v5.0RC1, as separate entries, and not just single v5.0. A lot of software do this and it looks good to me.
@sledgehammer999?

Reply:
I am thinking of promoting this to a `FEATURE` entry. "Add 'Protocol' column in trackers tab".

Reply:
> I am thinking of promoting this to a `FEATURE` entry. "Add 'Protocol' column in trackers tab".

It isn't a feature in itself. It is just a side effect of another changes.

Reply:
I opted to delete this entirely.
Bugfix changelog entries between unstable releases are good to have but hard to track.
I also believe they create noise on the main page. Interested parties can look at the git log.